maxim/logger/together/client.py

```
@@ -0,0 +1,174 @@
+_INSTRUMENTED = False
+
+def instrument_together(logger: Logger) -> None:
+    """Patch Together's chat completion methods for logging."""
+
+    global _INSTRUMENTED
+    if _INSTRUMENTED:
+        scribe().debug("[MaximSDK] Together already instrumented")
+        return
+
+
+    def wrap_sync_create(create_func):
```

Ruff (0.11.9)

`wrap_sync_create` is too complex (11 > 10)

(C901)